### ISSUE TITLE
Open setup wizard in the OS language

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#5832]
### Observed behavior
I thought we already had this issue filed at some point, but I'm unable to find it.

Post-install Kolibri setup wizard opens in English irregardless of the user's OS language. Since we advanced the work on the Kolibri default language setup, maybe this could be a good addition. Namely, the **setup wizard should open in the default language user has on their OS**, and default to English, if the language is not supported by Kolibri. 

This should be a cross-platform feature, but maybe in case of the Windows installer, if the user's OS language isn't supported we could set the one selected during the installation?

![setup-wizard-language](https://user-images.githubusercontent.com/1457929/62083710-e5dee600-b257-11e9-8ee1-79dbe3bdea8f.gif)

### Expected behavior
Better language-related UX?

### User-facing consequences
Less selections during the installation and setup proces?

### Errors and logs
…

### Steps to reproduce
…

### Context
…

**Comment by indirectlylit** [comment]
cc @kollivier created a task related to this for the new MacOS app: 
https://www.notion.so/learningequality/Set-up-internationalization-pipeline-32fe1c1084d14b4d8c3786ea0c69844a

**Comment by benjaoming** [comment]
I think we can try having the installer define a locale, and then Kolibri could choose this locale in the Setup Wizard if it's a supported language. Should be case insensitive and replace `_` with `-` when matching.

CC: @rtibbles - I think you've already had a look at the ways that language is chosen from HTTP request vs. `settings.LANGUAGE_CODE` etc..

**Comment by rtibbles** [comment]
We could currently do this by opening with the language code in the URL - I think this is unlikely to change, so might be a quick and stable way to move forward with this.

**Comment by benjaoming** [comment]
@rtibbles we would need a way to verify that Kolibri supports `<lang>` before opening up such a page... or to have the installer match up the platform language for the installer with a list of supported languages.

**Comment by rtibbles** [comment]
Good point - can check that the URL doesn't return a 404 (which is what it does for unrecognized lang codes).

**Comment by benjaoming** [comment]
@rtibbles -- good idea. I think that all the installer libraries would have an HTTP library to do a header request to a specific page, check if it's 404 and then default to _not_ setting a language code in the URL path if that's the case.
